front/src/modules/conflict/components/ConflictsListV2.tsx

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+
+import cx from 'classnames';
+import { useTranslation } from 'react-i18next';
+import { BsLightningFill } from 'react-icons/bs';
+
+import type { ConflictV2 } from 'common/api/osrdEditoastApi';
+import type { TrainScheduleWithDetails } from 'modules/trainschedule/components/TimetableV2/types';
+
+type ConflictTableProps = {
+  conflicts: ConflictV2[];
+  expanded: boolean;
+  trainSchedulesDetails: TrainScheduleWithDetails[];
+  toggleConflictsList: () => void;
+  onClick: (conflict: ConflictWithTrainNames) => void;
+};
+
+type ConflictWithTrainNames = ConflictV2 & {
+  train_names: string[];
+};
+
+function addTrainNamesToConflicts(
+  conflicts: ConflictV2[],
+  trainSchedulesDetails: TrainScheduleWithDetails[]
+): ConflictWithTrainNames[] {
+  const trainNameMap: { [id: number]: string } = {};
+
+  trainSchedulesDetails.forEach(({ id, trainName }) => {
+    trainNameMap[id] = trainName;
+  });
+
+  return conflicts.map((conflict) => ({
+    ...conflict,
+    train_names: conflict.train_ids.map((id) => trainNameMap[id] || ''),
+  }));
+}
+
+function string2time(sec: string) {
```

Comment:
I think it's better to move this function to `front/src/utils/date.ts` or create a new utils file in the same folder. Additionally, I believe it would be more consistent with our naming conventions to use `stringToTime ` instead of `string2time`.

Reply:
Nice catch ty, fixed